test/ApplePlatformsIntegrationTestApp/ApplePlatformsIntegrationTestMobileApp/AppDelegate.mm

```diff
@@ -0,0 +1,30 @@
+//
+//  AppDelegate.m
+//  ApplePlatformsIntegrationTestMobileApp
+//
+//  Created by Michał Grabowski on 17/08/2020.
+//
+
+#import "AppDelegate.h"
+
+#import <hermes/hermes.h>
+#import <jsi/jsi.h>
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    std::string s = "print('Hello world -- son of Maia and Zeus');";
+    std::string url = "http://example/js";
+    facebook::hermes::HermesRuntime::DebugFlags flags;
+    auto runtime = facebook::hermes::makeHermesRuntime();
+    runtime->debugJavaScript(s, url, flags);
+    exit(0);
+    return YES;
+}
```

Comment:
This is just a simple copy-paste from MacOS application. While it works, it is quite tricky to access console output (just like from a Mac app) and check its contents.

Any ideas here? I'd prefer to keep it simple and not go into a fully-fledged test suite. 

@alloy

Reply:
Good question. So currently you’re not doing any testing yet, right? Or did I overlook it?

Reply:
My only testing is printing this hello world string (just like on MacOS). It works, but I just realized it's not as easy to get the console output here. 

Possible ways to go forward:
- get the console output (there are some hacks)
- rewrite it to use XCTest or something similar

Reply:
Resolved by providing XCTest.